src/app/header/header.component.html

```diff
@@ -1,12 +1,21 @@
 <div class="header-cover" fxLayout="row" fxLayoutAlign="space-between">
   <div>Позашкілля</div>
-  <div *ngIf="isAuthorized; else isUnauthorized">
-    Увійти
-  </div>
+  <ng-container
+    *ngIf="(isAuthorized$ | async) then YesAuthorized; else NotAuthorized">
```

Comment:
is it more meaningful to use authorized and unauthorized instead of yes no prefixes?